cmd/podmanV2/pods/prune.go

```diff
@@ -0,0 +1,75 @@
+package pods
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/containers/libpod/cmd/podmanV2/registry"
+	"github.com/containers/libpod/cmd/podmanV2/utils"
+	"github.com/containers/libpod/pkg/domain/entities"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	pruneOptions = entities.PodPruneOptions{}
+)
+
+var (
+	pruneDescription = fmt.Sprintf(`podman pod prune Removes all exited pods`)
+
+	pruneCommand = &cobra.Command{
+		Use:     "prune [flags]",
+		Short:   "Remove all stopped pods and their containers",
+		Long:    pruneDescription,
+		RunE:    prune,
+		Example: `podman pod prune`,
+	}
+)
+
+func init() {
+	registry.Commands = append(registry.Commands, registry.CliCommand{
+		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
+		Command: pruneCommand,
+		Parent:  podCmd,
+	})
+	flags := pruneCommand.Flags()
+	flags.BoolVarP(&pruneOptions.Force, "force", "f", false, "Do not prompt for confirmation.  The default is false")
+}
+
+func prune(cmd *cobra.Command, args []string) error {
+	var (
+		errs utils.OutputErrors
+	)
+	if len(args) > 0 {
+		return errors.Errorf("`%s` takes no arguments", cmd.CommandPath())
+	}
+	if !pruneOptions.Force {
+		reader := bufio.NewReader(os.Stdin)
+		fmt.Println("WARNING! This will remove all volumes not used by at least one container.")
```

Comment:
"This will remove all volumes not used by at least one container"...looks like a copy&paste error.

Reply:
Upps. Fixed it.